### PR TITLE
created api redux app for activity log entries

### PR DESCRIPTION
**Pull request by FaithMo**
Related issues: [Create api redux apps for all activity log frontent components](Create api redux apps for all activity log frontent components)

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
pull in new changes from main to avoid possible conflicts?

**Review by Ngoakor12 — APPROVED** [review]

**Review by Vuyisanani — APPROVED** [review]

File: frontend/src/apiAccess/apiApps.js [hunk 1]
```diff
@@ -57,6 +57,8 @@ const FETCH_USER_ACTIONS_CARDS_COMPLETED_PAGE =
 
 const FETCH_ACTIVITY_LOG_DAY_COUNTS_PAGE = "FETCH_ACTIVITY_LOG_DAY_COUNTS_PAGE";
 
+const ACTIVITY_LOG_ENTRIES = "ACTIVITY_LOG_ENTRIES";
```
Comment (sheenarbw) [inline frontend/src/apiAccess/apiApps.js]: Is this meant to be a verb or a noun? Surely we are trying to fetch the activity log entries? This is the Action name, right?

File: frontend/src/apiAccess/apis.js [hunk 2]
```diff
@@ -379,6 +379,31 @@ async function activityLogDayCountsPage({
...
+  const limit = 20;
+  const offset = calculateOffset({ page, limit });
+  let params = {
+    limit,
+    offset,
+  };
+
+  if (eventTypeName) params["event_type__name"] = eventTypeName;
+  if (actorUser) params["actor_user"] = actorUser;
+  if (effectedUser) params["effected_user"] = effectedUser;
```
Comment (sheenarbw) [inline frontend/src/apiAccess/apis.js:403]: There are a few missing filters here: 

This is how it is set up in the backend
```
filterset_fields = [
        "event_type",
        "actor_user",
        "effected_user",
        "object_1_content_type",
        "object_1_id",
        "object_2_content_type",
        "object_2_id",
    ]
```

Ideally we should be able to make use of all the filters.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Some missing filters and a naming thing. Otherwise it's good.

File: frontend/src/apiAccess/apis.js [hunk 3]
```diff
@@ -379,6 +379,40 @@ async function activityLogDayCountsPage({
   return { response, responseData };
 }
 
+async function activityLogEntries({
```
Comment (Ngoakor12) [inline frontend/src/apiAccess/apis.js:384]: should the function name not also be updated? at the moment it seems to be missing a verb to show what the function is doing with the activity log entries.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

File: frontend/src/apiAccess/apis.js (hunk 3 above)
Comment (FaithMo) [reply frontend/src/apiAccess/apis.js:384]: i think the function name is okay, looking at the entire structure of the file

Comment (FaithMo) [reply frontend/src/apiAccess/apis.js:384]: i think the function name is okay, looking at the entire structure of the file

**Review by Ngoakor12 — APPROVED** [review]

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**